pygccxml/parser/scanner.py

```diff
@@ -186,6 +193,19 @@ def xml_generator_from_xml_file(self):
     def read(self):
         xml.sax.parse(self.xml_file, self)
 
+    def _handle_comment(self, declaration):
+        comm_decl = self.__declarations.get(declaration.comment)
+        if comm_decl:
+            with open(self.__files.get(comm_decl.location.file_name, "r")) as file:
+                line_list = file.readlines()
+                comment_text = []
+                for indx in range(comm_decl.start_line - 1,comm_decl.end_line):
+                    comm_line = line_list[indx]
+                    comm_line = comm_line.strip("\n")
```

Comment:
Consider replacing `.strip("\n")` with a more constrained `.rstrip("\n")` (only right side).

`.strip()` and its variants may remove multiple of the given characters, but I guess given that this is from readlines, it should be fine.